shm_wrapper/shm_wrapper.h

```diff
@@ -310,7 +325,8 @@ int gamepad_write(uint32_t pressed_buttons, float joystick_vals[4]);
  *    value: value of the parameter, with the corresponding type filled with data
  * Returns:
  *    0 on success
- *    -1 on failure (maximum number of custom parameters is reached so this parameter isn't written)
+ *   -1 when maximum number of custom parameters is reached
+ *   -2 when the given key is longer than LOG_KEY_LENGTH characters
```

Comment:
See my comment in `shm_wrapper.c`, but imo it might be clearer if it's phrased as
`...when the length of the key is longer than LOG_KEY_LENGTH` 
or if the null-terminator is involved, 
`...when the length of the key is longer than LOG_KEY_LENGTH - 1` 
C makes character count a bit annoying

Reply:
it should be the first one when you include the null byte in the count.